node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -352,6 +352,26 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       descr = "Path to data directory. Defaults to $HOME/.rnode"
     )
 
+    val lmdbMapSizeRspace = opt[Long](
+      required = false,
+      descr = "Map size (in bytes) for RSpace"
+    )
+
+    val lmdbMapSizeBlockdagstore = opt[Long](
+      required = false,
+      descr = "Map size (in bytes) for blockDAG store index"
+    )
+
+    val lmdbMapSizeBlockstore = opt[Long](
+      required = false,
+      descr = "Map size (in bytes) for block store index"
+    )
+
+    val lmdbMapSizeDeploystore = opt[Long](
+      required = false,
+      descr = "Map size (in bytes) for deploy storage"
+    )
```

Comment:
Is this map size config necessary?

Reply:
Forgot to tell you. Last time, we forgot about  `DeployStorage`which it  is now not KeyValueStore but some other implementation. I would fix this in another pr.